sdk/identity/Azure.Identity/src/DefaultAzureCredentialOptions.cs

```diff
@@ -6,36 +6,45 @@ namespace Azure.Identity
+        /// the cache the SharedTokenCacheCredential will not be used for authentication.
+        /// .</remarks>
+        public string SharedTokenCacheUsername { get; set; }
 
         /// <summary>
         /// Specifies the client id of the azure ManagedIdentity in the case of user assigned identity.
         /// </summary>
         public string ManagedIdentityClientId { get; set; }
 
         /// <summary>
-        /// Specifies whether the EnvironmentCredential will be included to potentially read authentication details from the process' environment variables.
+        /// Specifies whether the <see cref="EnvironmentCredential"/> will be excluded from the authentication flow. Setting to true disables reading
+        /// authentication details from the process' environment variables.
         /// </summary>
-        public bool IncludeEnvironmentCredential { get; set; } = true;
+        public bool ExcludeEnvironmentCredential { get; set; } = false;
 
         /// <summary>
-        /// Specifies whether the ManagedIdentityCredential will be included to potentially get tokens from the a managed identity endpoint if available.
+        /// Specifies whether the <see cref="ManagedIdentityCredential"/> will be excluded from the <see cref="DefaultAzureCredential"/> authentication flow.
+        /// Setting to true disables authenticating with managed identity endpoints.
         /// </summary>
-        public bool IncludeManagedIdentityCredential { get; set; } = true;
+        public bool ExcludeManagedIdentityCredential { get; set; } = false;
 
         /// <summary>
-        /// Specifies whether the SharedTokenCacheCredential will be included to potentially get tokens from the shared token cache if avaliable.
+        /// Specifies whether the <see cref="SharedTokenCacheCredential"/> will be excluded from the <see cref="DefaultAzureCredential"/> authentication flow.
+        /// Setting to true disables single sign on authentication with development tools which write to the shared token cache.
         /// </summary>
-        public bool IncludeSharedTokenCacheCredential { get; set; } = true;
+        public bool ExcludeSharedTokenCacheCredential { get; set; } = false;
 
         /// <summary>
-        /// Specifies whether the InteractiveBrowserCredential will be included to potentially get tokens by launching the system default browser to enable a user to log in.
+        /// Specifies whether the <see cref="InteractiveBrowserCredential"/> will be excluded from the <see cref="DefaultAzureCredential"/> authentication flow.
+        /// Setting to true disables launching the default system browser to authenitcate in developement environments.
         /// </summary>
-        public bool IncludeInteractiveBrowserCredential { get; set; } = false;
+        public bool ExcludeInteractiveBrowserCredential { get; set; } = false;
```

Comment:
I'm really not a fan of interactive on by default

Reply:
Nit - authenitcate => authenticate

Reply:
I especially don't like that we're turning it on so that API looks nice and not for other more substantial reasons.